### PR TITLE
Fix(sozo): Improve sozo migration output

### LINKED ISSUE #1558: Improve sozo migrate output
Currently there is a final `register_model` multicall which occurs at the end of a model migration. It isn't clear that this action relates to all models. We should improve the output to disambiguate it

```
# Models (6)
dope_world::paper_bridge::paper_bridge_model
  > Already declared: 0x92388c909a31858a75c6e766b4c956612bbc22d07dd7e6b94885e753503cdf
token::components::security::initializable::initializable_model
  > Already declared: 0x6782696aebf5bf50992a9e5f06d116b2f8bb37c8892618b3a975146681c900
token::components::token::erc20::erc20_allowance::erc_20_allowance_model
  > Already declared: 0x6fc168b12a303452bce6f90c4f0fc47699d8714f43aac3ea0a9cfeb55524e74
token::components::token::erc20::erc20_balance::erc_20_balance_model
  > Already declared: 0x682b4f16fe839e94a9700f750f5448142369273db0880dc89241f3dc183a253
token::components::token::erc20::erc20_bridgeable::erc_20_bridgeable_model
  > Already declared: 0x311a622c34bacc61d159bdc9656f16799839752bfd07f788bc706ecc3f36ec0
token::components::token::erc20::erc20_metadata::erc_20_metadata_model
  > Already declared: 0x18d6ed523121dce641165ea669cd427e285c59a3e12b1625602dbe525bbcfe4
  > Registered at: 0x16363b4399c361e8817be0776c03e2bba20a99a5b49a246231021f40798cfae
  ```

### PR DESCRIPTION
**Pull request by hhamud**
- Fixes  #1558 
Added an additional divider line between the initial messages and final model registration message to make it clearer.

### CODE REVIEW

File: bin/sozo/src/ops/migration/mod.rs [hunk 1]
```diff
@@ -486,7 +486,9 @@ where
 
     TransactionWaiter::new(transaction_hash, migrator.provider()).await?;
 
-    ui.print_sub(format!("Registered at: {transaction_hash:#x}"));
+    ui.print_header(format!("======================================================"));
+
+    ui.print_sub(format!("All models are registered at: {transaction_hash:#x}"));
```
Comment (glihm) [inline bin/sozo/src/ops/migration/mod.rs:491]: Could we change that for a simple print? To. It have it indented as the model declaration?

File: bin/sozo/src/ops/migration/mod.rs [hunk 2]
```diff
@@ -486,7 +486,9 @@ where
 
     TransactionWaiter::new(transaction_hash, migrator.provider()).await?;
 
-    ui.print_sub(format!("Registered at: {transaction_hash:#x}"));
+    ui.print_header(format!("======================================================"));
```
Comment (glihm) [inline bin/sozo/src/ops/migration/mod.rs:489]: Also here it should not be a header as it should still under models. Perhaps only having an empty line to make the separation is good, or at least remove header.

**Review by glihm — CHANGES_REQUESTED** [review]
Hey thanks your for that sorry for the delay. Let me know if you have questions on the comments! 👍

File: bin/sozo/src/ops/migration/mod.rs (hunk 1 above)
Comment (hhamud) [reply bin/sozo/src/ops/migration/mod.rs:491]: do you mean line 491? or 489?

File: bin/sozo/src/ops/migration/mod.rs (hunk 2 above)
Comment (hhamud) [reply bin/sozo/src/ops/migration/mod.rs:489]: I thought adding some sort of divider would make the delineation that much clearer. I will remove it if it doesn't.

**Review by glihm — APPROVED** [review]
Thanks for the update!

image

Looks good to me, or @tarrencev did you had more changes in mind with this issue?

**Pull request merged**